src/modules/git_status.rs

```diff
@@ -114,7 +115,15 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 
     // Add the stashed segment
     if stash_object.is_ok() {
-        module.create_segment("stashed", &config.stashed);
```

Comment:
Do we still need the `stash_object` variable (used one line above)?

Reply:
It doesn't appear so. Without it all tests pass and the stash module seems unaffected overall.